decidim-forms/app/commands/decidim/forms/answer_questionnaire.rb

```diff
@@ -43,15 +43,15 @@ def call
       # of this problem.
       def reset_form_attachments
         @form.responses.each do |answer|
-          answer.errors.add(:add_documents, :needs_to_be_reattached) if answer.has_attachments?
+          answer.errors.add(:add_documents, :needs_to_be_reattached) if answer.has_attachments? || answer.has_error_in_attachments?
```

Comment:
do we need the "or" or we can only check if the answer `has_error_in_attachments?`

Reply:
`has_attachments?` checks if question has attachments and if there is an `add_document` errors, maybe if is another error instead `add_documents` related with attachments, that condition can check it.

Reply:
ok, it's ok like this then